src/server/common/helpers/auth/federated-oidc.js

```diff
@@ -0,0 +1,239 @@
+import * as Hoek from '@hapi/hoek'
```

Comment:
feels like all this work is crying out for some tests

Reply:
Yeah, though once we've mocked out openid-client, cognito and yar, there's not a lot of non-mock logic to test.
I'd be cautious about the mock heavy tests giving too much false confidence.